### ISSUE TITLE
Sync with AntennaPod fails

### ISSUE THREAD

**Issue by update-freak** [SamTV12345/PodFetch#1055]
### Describe the bug

I currently have the issue with latest AntennaPod (v3.5.0) and PodFetch (v4.7.1) that the sync is not working anymore.
I also created from a clean setup but the sync still fails.

services:
  podfetch:
    image: samuel19982/podfetch:latest
    container_name: PodFetch
    user: 1026:100
    ports:
      - 4135:8000
    volumes:
      - /volume1/docker/podfetch/podcasts:/app/podcasts:rw
      - /volume1/docker/podfetch/db:/app/db:rw
    environment:
      - POLLING_INTERVAL=300
      - SERVER_URL=https://podfetch.beispiel.dynv6.net
      - DATABASE_URL=sqlite:///app/db/podcast.db
      - BASIC_AUTH=true
      - USERNAME=Felix
      - PASSWORD=d34gtfcw4g2fv
      - GPODDER_INTEGRATION_ENABLED=true
    restart: unless-stopped

1. Adding a user: docker exec -it DOCKER-ID /app/podfetch users add
(here I added the user felix with the role user) (https://github.com/SamTV12345/PodFetch/issues/523)
2. Restart the container
3. Login with the Felix-User
4. Add all podcasts from AntennaPod
5. Logout with Felix and login with felix
6. PodFetch -> Profil -> API-Key ► generate ► save (https://github.com/SamTV12345/PodFetch/issues/544#issuecomment-1890400420)

Do you see any mistakes?

### Reproduction

see description

### System Info

```Shell
Synology NAS DSM 7.2 with Docker
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/SamTV12345/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/SamTV12345/.github/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by morpheus302** [comment]
The update fixed it for me but i dont see any device under "Manage Gpodder Podcasts"
![image](https://github.com/user-attachments/assets/db067dd3-7433-4dce-a8b2-9d0c9bb91ef2)

**Comment by update-freak** [comment]
perfect. Thank you very much for this quick fix :)

**Comment by update-freak** [comment]
In Step 3. (see initial post) I can not login with my standard user form the docker compose. 
Therefore I can not add the odcasts from the standarduser Felix in Step 4 .
![pic1](https://github.com/user-attachments/assets/f53d803e-131f-4139-b4be-ff8caa7f81ed).

The other gpodder user felix is able to login

I tried to sync between my devices, but only the subscriptions are synced and not the podcasts episodes which I finished to listen.
@SamTV12345 Is there any mistake in my steps or a bug regarding finished listen status of the episodes?
